### PR TITLE
Use apt without looping

### PR DESCRIPTION
**Pull request by mairas**
The "with_items" approach caused some deprecation warnings - this PR uses the approach given in the Ansible documentation.

### CODE REVIEW

**Comment by chacal** [comment]
50a42e06c41dffb405c46dc80ef9cb35811baac3 looks good. 👍 Two other commits are already included in https://github.com/tkurki/marinepi-provisioning/pull/55, yes? Maybe we should merge that first and rebase this on top of the new master afterwards?

**Comment by mairas** [comment]
Ah yes, intended to make all PRs independent but was a bit sloppy. Let's work from the oldest onwards.

File: roles/pysk/tasks/main.yml [hunk 1]
```diff
@@ -14,10 +14,10 @@
 ---
 - name: 'Install pysk dependencies'
   apt:
-    pkg: '{{item}}'
+    name: '{{ packages }}'
     state: latest
-  become: true
```
Comment (chacal) [inline roles/pysk/tasks/main.yml]: We could maybe keep this `become: true` here as it doesn't really have anything to do with the apt changes?

Comment (mairas) [reply roles/pysk/tasks/main.yml]: It's kinda redundant, innit? The top-level playbook file has `become: yes`, and the practice seems to be to set `become: false` in those roles/tasks that don't expect it.

But if you strongly prefer, I've no problem adding it back, of course.

Comment (chacal) [reply roles/pysk/tasks/main.yml]: Yep, most roles practically assume that the whole playbook is run as `become: true` and I think it is fine. This just caught my eye as it looks like it just slipped away (e.g. the other task in the same role still has it).

I'd say that either let's add it back or remove `become` from the other task as well to leave the role in a consistent state. :)

Comment (mairas) [reply roles/pysk/tasks/main.yml]: I'll add it back, better deal with the becomes in a separate PR (or at least a commit).

**Comment by chacal** [comment]
Looks good, thanks! 👍

**Pull request merged**